### PR TITLE
bump NN to 123.3.1, Common to 23.2.2, Maps to 10.10.1

### PR DESCRIPTION
**Pull request by dzinad**

### CODE REVIEW

File: changelog/unreleased/features/6886.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Added custom header x-mapbox-app-info to requests.
```
Comment (LukasPaczos) [inline changelog/unreleased/features/6886.md:1]: Is this relevant for developers? I think we can skip it.

File: changelog/unreleased/features/6886.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+- Fix upcoming route alerts incorrect position.
```
Comment (LukasPaczos) [inline changelog/unreleased/features/6886.md:2]:
```suggestion
- Fixed an issue with incorrect position of  `UpcomingRoadObject`s.
```
Is that true for all objects or only the ones that are not on the first leg? If only multi-leg routes were impacted, let's specify.

File: changelog/unreleased/features/6886.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+- Stabilized routes with alternatives refresh / re-route on deviate.
```
Comment (LukasPaczos) [inline changelog/unreleased/features/6886.md:3]: What does it mean "stabilized", could we get more specific on what issue was resolved?

File: changelog/unreleased/features/6886.md (hunk 1 above)
Comment (dzinad) [reply changelog/unreleased/features/6886.md:1]: Wasn't this what our customer requested? I mean in the `change-management/issues/1222` "customer impact" is filled.

Comment (LukasPaczos) [reply changelog/unreleased/features/6886.md:1]: My understanding is that this is an internal feature, customers will not need to interact with these headers directly.

Comment (dzinad) [reply changelog/unreleased/features/6886.md]: Removed this entry.

File: changelog/unreleased/features/6886.md (hunk 3 above)
Comment (dzinad) [reply changelog/unreleased/features/6886.md]: Removed this entry.

File: changelog/unreleased/features/6886.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+- Started using separate DeadReckoningDetector for Garmin GNSS receiver which uses accuracy for better tunnel exit detection.
+- Fixed situations with intensive re-routes on looped/u-turn routes by changing the route matcher's choice of the first location. Now we use fallback localization controller matching to provide the first active guidance progress after setting a new route to help the route matcher start working from the right segment (it already takes road direction, bearing, and looped routes into account).
+- Set limit of simultaneously running onboard route requests in config to 2 to prevent excessive resource consumption during intensive/repeated re-routes.
+- Fixed potential timestamp inconsistencies that may have led to location rejection by making the timestamp shift corrector configurable and disabled by default.
+- Fixed an issue with incorrect `UpcomingRoadObject#distanceToStart` values on the second, third, etc. legs.
```
Comment (LukasPaczos) [inline changelog/unreleased/features/6886.md:5]:
```suggestion
- Fixed an issue with incorrect `UpcomingRoadObject#distanceToStart` values on multi-leg routes.
```
Can be adjusted when building final changelog.

**Review by LukasPaczos — APPROVED** [review]

**Pull request merged**